examples/src/main/java/com/expediagroup/sdk/xap/examples/scenarios/car/CarListingsQuickStartScenario.java

```diff
@@ -33,140 +33,160 @@
  */
 public class CarListingsQuickStartScenario implements XapScenario {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(CarListingsQuickStartScenario.class);
```

Comment:
Can we please undo the indentation changes to focus on the code changes?

Reply:
Auto-formatted the code.